app-admin/etcd-wrapper/files/etcd-wrapper.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=etcd (System Application Container)
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
```

Comment:
Let's add an empty `Environment=ETCD_TAG=` so it's obvious that it needs to be overridden.